### ISSUE TITLE
The news/articles carousel is missing controls

### ISSUE THREAD

**Issue by mauromsl** [BirkbeckCTP/clean#13]
The carousel shown on the homepage rotates its items periodically. Users should be allowed to control its movement by pausing and/or skiping forwards and backwards.

WCAG 2.2.2
